cms/templates/partials/catalog-card.html

```diff
@@ -0,0 +1,89 @@
+        <div class="cover-image">
+            <div class="cover-image-frame">
+                {% if courseware_page.thumbnail_image %}
+                    {% image courseware_page.thumbnail_image fill-335x203 %}
+                {% else %}
+                    <img src="{% static default_image_path %}" alt="Preview image">
+                {% endif %}
+            </div>
+        </div>
+        <div class="top-level-details">
+            <h3 class="title">
+                    <a href="{% pageurl courseware_page %}">
+                        {{ courseware_page.title }}
+                    </a>
+            </h3>
+            <ul>
+                {% if courseware_page.product.current_price %}
+                    <li>
+                        <strong>Cost:</strong>
+                        ${{ courseware_page.product.current_price|floatformat:"0"|intcomma }}
+                    </li>
+                {% endif %}
+                {% if object_type == "program" %}
+                    <li>
+                        <strong>Number of Courses:</strong>
+                        {{ courseware_page.product.courses.count }}
+                    </li>
+                {% endif %}
+                {% if courseware_page.duration %}
+                    <li>
+                        <strong>Duration:</strong>
+                        {{ courseware_page.duration }}
+                    </li>
+                {% endif %}
+                {% if courseware_page.product.next_run_date %}
+                    <li>
+                        <strong>Next Start Date:</strong>
+                        {{ courseware_page.product.next_run_date|date:"F j, Y" }}
+                    </li>
+                {% endif %}
+            </ul>
+        </div>
+        <div class="offering-type">
+            <label>
+                {% if object_type == "course" %}
+                    Single
+                {% else %}
+                    Multi
+                {% endif %}
+                Course Online
+            </label>
+        </div>
+    </div>
+    <div class="bottom">
+      <input type="checkbox" class="toggle" id="collapsible_{{ tab }}_{{ object_type }}_{{ courseware_page.pk }}" data-target="#collapse-{{ tab }}-{{ object_type }}-{{ courseware_page.pk }}" data-toggle="collapse" />
+      <label for="collapsible_{{ tab }}_{{ object_type }}_{{ courseware_page.pk }}" class="lbl-toggle">
+          Details
+      </label>
+      <div id="collapse-{{ tab }}-{{ object_type }}-{{ courseware_page.pk }}" class="collapse">
+        <div class="details-body">
+          {% if courseware_page.description %}
+            <div class="description">{{ courseware_page.description|richtext }}</div>
+          {% endif %}
+          {% if object_type == "program" %}
+            <ul class="program-course-links">
+              {% for course_page in courseware_page.course_pages %}
+                <li>
+                  <a href="{% pageurl course_page %}" target="_blank">
```

Comment:
Was there a requirement that these links open in a new tab? Same comment for the link at line 82

Reply:
Not really, it was requested by @asadiqbal08 do you want me to remove it?

Reply:
NVM, I just went ahead and removed it, was creating too many tabs anyway.